primeqa/ir/dense/dpr_top/dpr/dpr_util.py

```diff
@@ -9,17 +9,11 @@
 class DPROptions(HypersBase):
     def __init__(self):
         super().__init__()
-        self.corpus_endpoint = ''
         self.port = 5001
-        self.model_name_or_path = ''  # used as qry_encoder_path
         self.rag_model_path = ''
```

Comment:
Are we running RAG ? I thought we just do DPR..

Reply:
Good point, removed `self.rag_model_path = ''`